### ISSUE TITLE
How to generate the collection ca and server client key when I deploy on real server such as amazon

### ISSUE THREAD

**Issue by kieugol** [gbahamondezc/node-grpc-ssl#1]
Hi Mr @gbahamondezc 
Im facing with issue when deploying on real server like amazon.

Im trying to generate  the collection key as ur introduction below:
https://github.com/gbahamondezc/node-grpc-ssl/blob/master/scripts/gen_certs.sh
I also changed localhost to my domain.
But It doesnt work when I deployed on real server

**Comment by giri-jeedigunta** [comment]
Yep having the same issue. I have used the script to generate the certificates these work fine locally but doesn't work as expected on dev or other environments. @kieugol what was your work around?
